src/java/org/apache/cassandra/cql3/statements/schema/DropFunctionStatement.java

```diff
@@ -120,6 +120,14 @@ public Keyspaces apply(Keyspaces schema)
         if (!dependentAggregates.isEmpty())
             throw ire("Function '%s' is still referenced by aggregates %s", name, dependentAggregates);
 
+        String dependentTables =
```

Comment:
formatting

Reply:
I've copied the aligning of `dependentAggregates` right above. Do you want me to change it to:
```java
        String dependentTables = keyspace.tablesUsingFunction(function)
                                         .map(table -> table.name)
                                         .collect(joining(", "));
```
?

Reply:
I would change it to the first option you mention, I don't see any reason not to unless I'm missing sthg.